zk.js/src/utxo.ts

```diff
@@ -524,6 +524,8 @@ export class Utxo {
    * @returns {string}
    */
   getCommitment(poseidon: any): string {
+    // TODO: UNDO!!!
+    this._commitment = undefined;
```

Comment:
let's undo, since we adapted the other code in generate tx integrity hash it should be ok now.